github/projects.go

```diff
@@ -19,15 +19,18 @@ type ProjectsService service
 
 // Project represents a GitHub Project.
 type Project struct {
-	ID        *int64     `json:"id,omitempty"`
-	URL       *string    `json:"url,omitempty"`
-	OwnerURL  *string    `json:"owner_url,omitempty"`
-	Name      *string    `json:"name,omitempty"`
-	Body      *string    `json:"body,omitempty"`
-	Number    *int       `json:"number,omitempty"`
-	CreatedAt *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt *Timestamp `json:"updated_at,omitempty"`
-	NodeID    *string    `json:"node_id,omitempty"`
+	ID         *int64     `json:"id,omitempty"`
+	URL        *string    `json:"url,omitempty"`
+	HtmlURL    *string    `json:"html_url,omitempty"`
```

Comment:
Let's be consistent with the rest of the naming in this repo and call this `HTMLURL` (for better or worse).